solr/test-framework/src/java/org/apache/solr/cloud/ChaosMonkey.java

```diff
@@ -149,8 +152,7 @@ public void expireSession(final JettySolrRunner jetty) {
     if (cores != null) {
       monkeyLog("expire session for " + jetty.getLocalPort() + " !");
       causeConnectionLoss(jetty);
-      long sessionId = cores.getZkController().getZkClient().getSolrZooKeeper().getSessionId();
-      zkServer.expire(sessionId);
+      cores.getZkController().getZkClient().getZooKeeper().getTestable().injectSessionExpiration();
```

Comment:
do we not need to expire on the server as well?

Reply:
Nope! All the server does on expire is send the expiration packet.

Reply:
Mike and I have determined that we do need to expire on the server still, so that ephemeral nodes don't take 30 seconds (the default `zkClientTimeout`) to be deleted.

Reply:
reverted in b386e144e2f